src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.tsx

```diff
@@ -42,15 +66,80 @@ export const KafkaDetailsTab: FunctionComponent<KafkaDetailsTabProps> = ({
       <DetailsTabAlert expiryDate={expiryDate} />
       <TextContent>
         <TextList component={TextListVariants.dl}>
+          {isTesting && (
+            <>
+              {renderTextListItem(
+                t("common:type"),
+                instanceType === "eval"
+                  ? t("common:trial")
+                  : t("common:standard")
+              )}
+              {instanceType === "standard" &&
+                renderTextListItem(
+                  t("size"),
+                  t("create-kafka-instance:streaming_size_value", {
+                    value: size,
+                  })
+                )}
+              {renderTextListItem(
+                t("ingress"),
+                t("create-kafka-instance:ingress_value", {
+                  value: ingress,
+                })
+              )}
+              {renderTextListItem(
+                t("egress"),
+                t("create-kafka-instance:egress_value", {
+                  value: egress,
+                })
+              )}
+              {renderTextListItem(
+                t("storage"),
+                t("create-kafka-instance:storage_value", {
+                  value: storage,
+                })
+              )}
+              {renderTextListItem(
+                t("partitions"),
+                t("create-kafka-instance:partitions_value", {
+                  value: maxPartitions,
+                })
+              )}
+              {renderTextListItem(
+                t("client_connections"),
+                t("create-kafka-instance:client_connections_value", {
+                  value: connections,
+                })
+              )}
+              {renderTextListItem(
+                t("connection_rate"),
+                t("create-kafka-instance:connection_rate_value", {
+                  value: connectionRate,
+                })
+              )}
+              {renderTextListItem(
+                t("message_size"),
+                t("create-kafka-instance:message_size_value", {
+                  value: messageSize,
+                })
+              )}
+            </>
+          )}
+          {renderTextListItem(t("common:id"), id)}
+          {renderTextListItem(t("common:owner"), owner)}
+          {renderTextListItem(
+            t("common:time_created"),
+            format(createdAt, "PPPP,p")
+          )}
+          {renderTextListItem(
+            t("common:time_updated"),
+            format(updatedAt, "PPPP,p")
+          )}
```

Comment:
Currently in console.redhat.com, these values display as:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/21063328/158699866-809272b1-af90-4c73-8ac2-8b242f08201d.png">

But the example in storybook for this PR shows this:
![image](https://user-images.githubusercontent.com/21063328/158700105-6e1f95b5-a460-4c76-8dd0-16c422abf32d.png)

The property labels are what I provided in the design, but another difference between these two is the way the date and time values are combined.

Currently we use just a space `...2022 6:00 PM`, but this update replaces the space with a comma not followed by a space`...2022,6:00PM`

Now that I see the comma, it does make sense to include it, but can you add a space after the comma so that it displays like this instead?:
> Tuesday, March 8, 2022, 6:43 AM

Reply:
done